### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Daniielmayo**
unificación de ramas

### CODE REVIEW

**Review by abdulflorez — APPROVED** [review]

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]
Dani, I letf some comments about the px pending to be changed... just take note to open a techDebt... The first comment is about `.env` file, I think it should be on our `.gitignore` file

File: .env
[the remark this answers is not in the document]
Comment (abdulflorez) [reply .env:1]: @DanielCalleCO Yes, but after completing the task related to  the .env, the idea is that the workflow generates it automatically, once that is done part of that task should be added to the .gitignore.

File: components/atoms/Input/Input.module.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  position: relative;
+  padding: 0.6rem;
+  margin-top: 0.8rem;
+  display: flex;
+  align-items: center;
+}
+
+.password-toggle-button {
+  position: absolute;
+  right: 20px;
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/atoms/Input/Input.module.css:11]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: components/atoms/icons/errorRegisterIcons/ErrorRegisterIcon.module.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.error-register-icon {
+  color: var(--darkGray);
+  width: 1.3rem;
+  margin: 0 0.6rem;
+  background: var(--white);
+  border-radius: 50%;
+  border: 1px solid var(--darkGray);
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/atoms/icons/errorRegisterIcons/ErrorRegisterIcon.module.css:7]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: components/organisms/ChangePasswordUser/ChangePasswordUser.module.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+.container-arrow-back {
+  width: 100%;
+}
+.link-backProfile {
+  display: flex;
+  align-items: center;
+  margin-bottom: 30px;
+  gap: 30px;
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/organisms/ChangePasswordUser/ChangePasswordUser.module.css:32]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: components/organisms/ChangePasswordUser/ChangePasswordUser.module.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  align-items: center;
+}
+
+.container-arrow-back {
+  width: 100%;
+}
+.link-backProfile {
+  display: flex;
+  align-items: center;
+  margin-bottom: 30px;
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/organisms/ChangePasswordUser/ChangePasswordUser.module.css:31]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: components/organisms/Modal/Modal.module.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  padding: 2.4rem;
+}
+
+.modalEditProfile-container {
+  position: relative;
+  width: 23rem;
+  padding: 1.2rem;
+  background: var(--white);
+  border-radius: 0.8rem;
+  box-shadow: #64646f33 0px 7px 29px 0px;
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/organisms/Modal/Modal.module.css:20]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: components/organisms/Modal/Modal.module.css [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  transition: 0.3s ease all;
+  color: var(--darkBlue);
+}
+
+.modalEditProfile-iconClose:hover {
+  border-radius: 0.5rem;
+  color: var(--turquoise);
+  background-color: var(--transparentBackground);
+}
+@media (width >= 1024px) {
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/organisms/Modal/Modal.module.css:45]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: components/organisms/Modal/Modal.module.css [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+  .modalEditProfile-container {
+    width: 600px;
```
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply components/organisms/Modal/Modal.module.css:47]: Pending to be resolved in the next sprint. [Here](https://danielcalleco.atlassian.net/browse/DES-221) is the ticket

File: .env
[the remark this answers is not in the document]
Comment (DanielCalleCO) [reply .env:1]: [Ticket](https://danielcalleco.atlassian.net/browse/DES-222) opened to resolve on the next sprint. Thanks Dany

**Review by DanielCalleCO — APPROVED** [review]

**Review by SthefaniaEcheverry — APPROVED** [review]

**Pull request merged**